### PR TITLE
Edits

### PR DESCRIPTION
**Pull request by waynebruce0x**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/nord-finance exports TVL: 
        
 
 
    avalanche-staking         18.61 k
    ethereum                  200.38 k
    avalanche                 705.56 k
    polygon-pool2             0
    ethereum-staking          747.18 k
    polygon                   39.21 k
    ethereum-pool2            332.210856
    polygon-staking           619.69 k
    avalanche-pool2           0
    staking                   1.39 M
    pool2                     332.210856
    
    total                    945.16 k

**Comment by llamatester** [comment]
The adapter at projects/qidao exports TVL: 
        
 
 
    xdai                      9.44 k
    polygon                   147.61 M
    harmony                   229.26 k
    avalanche                 4.66 M
    moonriver                 15.82 k
    fantom                    222.08 M
    
    total                    374.60 M

**Comment by llamatester** [comment]
The adapter at projects/nord-finance exports TVL: 
        
 
 
    avalanche-staking         18.63 k
    ethereum                  199.55 k
    ethereum-pool2            331.81701452899995
    ethereum-staking          747.93 k
    avalanche                 706.24 k
    polygon-staking           620.31 k
    avalanche-pool2           0
    polygon-pool2             0
    polygon                   38.16 k
    staking                   1.39 M
    pool2                     331.81701452899995
    
    total                    943.95 k

**Comment by llamatester** [comment]
The adapter at projects/qidao exports TVL: 
        
 
 
    xdai                      9.34 k
    avalanche                 4.62 M
    harmony                   225.19 k
    moonriver                 15.65 k
    polygon                   146.88 M
    fantom                    221.00 M
    
    total                    372.75 M

File: projects/burrow.cash/index.js [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+    return balances;
+  };
+};
 
 module.exports = {
   near: {
-    tvl,
+    tvl: tvl(),
+    borrowed: tvl(true)
   },
```
Comment (g1nt0ki) [inline projects/burrow.cash/index.js:25]: can you add flag `timetravel: false` here

File: projects/qidao/index.js [hunk 2]
```diff
@@ -364,6 +365,31 @@ async function fantom(timestamp, block, chainBlocks) {
...
+    'fantom',
+    transformAddress),
+    handleYearnTokens(balances, [
+      "0xd817A100AB8A29fE3DBd925c2EB489D67F758DA9",
+    ], 
+    "0x571F42886C31f9b769ad243e81D06D0D144BE7B4",
+    chainBlocks.fantom,
+    'fantom',
+    transformAddress),
+    handleYearnTokens(balances, [
```
Comment (g1nt0ki) [inline projects/qidao/index.js:384]: I didnt know about this helper, good to know

**Review by g1nt0ki — APPROVED** [review]

**Comment by llamatester** [comment]
The adapter at projects/nord-finance exports TVL: 
        
 
 
    ethereum                  208.34 k
    avalanche-pool2           0
    polygon                   44.32 k
    ethereum-pool2            335.327897
    avalanche                 705.75 k
    avalanche-staking         18.61 k
    ethereum-staking          747.67 k
    polygon-staking           620.33 k
    polygon-pool2             0
    pool2                     335.327897
    staking                   1.39 M
    
    total                    958.42 k

**Comment by llamatester** [comment]
The adapter at projects/qidao exports TVL: 
        
 
 
    xdai                      9.36 k
    avalanche                 4.63 M
    moonriver                 15.84 k
    polygon                   146.18 M
    harmony                   219.25 k
    fantom                    223.07 M
    
    total                    374.13 M

**Pull request merged**